app/presenters/dependants_presenter.rb

```diff
@@ -19,19 +19,19 @@ def dependants_by_age_range
 
     sorted_dependants.each do |dependant|
       case dependant[:age]
-      when 0, 1
+      when 0
         grouped_dependants['0 to 1'] += 1
-      when 2, 3, 4
+      when 1, 2, 3
```

Comment:
Wondering whether useful to put in a comment somewhere here to remind people that the copy refers to 'next birthday'